.github/actions/e2e-prepare-containers/action.yml

```diff
@@ -62,7 +62,7 @@ runs:
             --env LDAP_PASSWORDS=123456,123465 \
             --env LDAP_ROOT=dc=example,dc=org \
             --env LDAP_PORT_NUMBER=389 \
-            bitnami/openldap:2.6.4 &&
+            bitnamilegacy/openldap:2.6.4 &&
```

Comment:
Even though I just backport https://github.com/metabase/metabase/pull/64264, I don't wanna have to change the whole thing here to docker compose that was change in https://github.com/metabase/metabase/pull/60125, but not backported (so only available from 56+)